Flagsmith.FlagsmithClient/Cache/IdentityTraitsKey.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Flagsmith
+{
+    public class IdentityTraitsKey
+    {
+        public string Identity { get; set; }
+        public List<ITrait> Traits { get; set; }
+
+        public IdentityTraitsKey(string identity, List<ITrait> traits)
```

Comment:
We should use `identifier` whenever we're talking about the string key that represents an identity.

Reply:
will fix that shortly